learn/pipelines/index.qmd

```diff
@@ -0,0 +1,69 @@
+---
+title: "Outbreak analytics Pipelines"
+author:
+  - name: "Andree Valle-Campos"
+    orcid: "0000-0002-7779-481X"
+  - name: " Carmen Tamayo Cuartero"
+    orcid: "0000-0003-4184-2864"
+  - name: "Anna Carnegie"
+    orcid: "0000-0002-6385-7795"
+  - name: "Sebastian Funk"
+    orcid: "0000-0002-2842-3406"
+  - name: "Adam Kucharski"
+    orcid: "0000-0001-8814-9421"
+  - name: "Rosalind M Eggo"
+    orcid: "0000-0002-0362-6717"
+date: last-modified
```

Comment:
Nice! I didn't know about this!

Reply:
possibly I spend too much time looking at the quarto documentation, hehe

Reply:
I now realize this may come with issues as we sometimes need to update already published posts (typos, breaking changes in quarto, broken URLs, etc.)